build_pbf_glyphs/src/main.rs

```diff
@@ -76,11 +80,13 @@ async fn combine_glyphs(font_path: &Path, font_names: &[&str], stack_name: Strin
         // The above utility always returns a single stack
         glyphs_combined += stack.stacks[0].glyphs.len();
 
+        let encoded_bytes = stack.encode_to_vec();
```

Comment:
This looks suspiciously heavy for `async`.

Reply:
Ahh, fair point. It's not actually an issue in the current usage (this isn't the similar library function but an internal helper at the end of build_pbf_glyphs) but yeah this probably should be on a blocking thread generally. Fixed